projects/ngx-piwik-pro/src/lib/services/cookie-management/cookie-management.service.ts

```diff
@@ -1,74 +1,43 @@
+import { CookieManagement } from '@piwikpro/tracking-base-library'
 import { Injectable } from '@angular/core';
-import { PaqService } from '../../services/paq/paq.service';
-import { TRACK_EVENT } from '../../constants/track-event.constant';
+
+type ICookieManagement = typeof CookieManagement;
 
 @Injectable({
   providedIn: 'root'
 })
 export class CookieManagementService {
-
-  constructor(
-    private readonly paqService: PaqService
-  ) {}
-
-  disableCookies() {
-    this.paqService.push([TRACK_EVENT.DISABLE_COOKIES]);
-
+  disableCookies(...params: Parameters<ICookieManagement['disableCookies']>) {
+    CookieManagement.disableCookies(...params);
   }
-
-  enableCookies() {
-    this.paqService.push([TRACK_EVENT.ENABLE_COOKIES]);
-
+  enableCookies(...params: Parameters<ICookieManagement['enableCookies']>) {
```

Comment:
Praise: Thanks for passing the params for the functions that do not take any arguments right now. They may in the future and we don't have to remember to update them :pray: